### DISCUSSION TITLE
Still waiting to receive academic benefits

### DISCUSSION THREAD

**Discussion by stanleyycheung** [community/community#142171] [GitHub Education]
### Hi everyone,

Hi Github team, I am still waiting for the academic benefits after being approved last week. Do you know when should I expect to receive them?

**Comment by luannndev** [comment]
Hi stanleyycheung,

It's great to hear that you've been approved for GitHub's academic benefits! However, it can sometimes take a little while for the benefits to be processed and fully activated after approval. Typically, you should expect to receive access within a few days to a week, but this can vary.

If you haven't received your benefits after a week, I recommend reaching out to GitHub Education support directly for more specific information regarding your account status. They should be able to provide you with an update on when you can expect your benefits to be available.

In the meantime, keep an eye on your email for any updates or confirmations from GitHub.

I hope you receive your benefits soon!

**Comment by davevad93** [comment]
Hi @stanleyycheung ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @stanleyycheung was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
